src/db.rs

```diff
@@ -1027,6 +1027,18 @@ impl Db {
 	pub fn stats(&self) -> StatSummary {
 		self.inner.stats()
 	}
+
+	/// Add a new column with options specified by `new_column_options`.
+	pub fn add_column(options: &mut Options, new_column_options: ColumnOptions) -> Result<()> {
+		options.columns.push(new_column_options);
+		options.write_metadata_with_version(
+			&options.path,
+			&options.salt.expect("Salt is required to add column"),
```

Comment:
Should not be required really. When opening the database `None` means that existing value must be used. After you open the database as per previous comment, you can get the existing salt value.

Reply:
done!